src/projectPath.ts

```diff
@@ -0,0 +1,6 @@
+import * as path from "path";
+import * as detectProjectRoot from "./detectProjectRoot";
+
+export function resolveProjectPath(cwd: string, filePath: string): string {
```

Comment:
Could you add some documentation for this? I'd just like to know the answer to "why would I use this?"

Reply:
For sure - good catch.

Reply:
Added in the latest commit